xbmc/guilib/GUIEditControl.cpp

```diff
@@ -565,7 +565,7 @@ bool CGUIEditControl::SetStyledText(const std::wstring &text)
   // show the cursor
   unsigned int ch = L'|';
   if ((++m_cursorBlink % 64) > 32)
-    ch |= (3 << 16);
+    ch = L' ';
```

Comment:
Can you explain this change?

Reply:
For the blinking effect, the cursor will alternate between '|' and ' ' (simple space - 0x20) instead of between '|' and '𰁼' (It does look like a space in some fonts - 0x3007C)

The '𰁼' while looking like a space (or a double space), seem to reverse Arabic words around it even in word processors (If the paragraph direction is LTR).

![Space VS 0x3007C](https://user-images.githubusercontent.com/63805744/80004225-5c94d100-84c2-11ea-906e-fe06444f77f1.png)

Reply:
Great, thanks. Your explanation makes sense.

Although I’m not sure I understand why that was chosen for the original code! A mystery.